### ISSUE TITLE
Disaster : discrimination huge bug with heavy ads and crypto mining

### ISSUE THREAD

**Issue by trimechee** [ungoogled-software/ungoogled-chromium#2066]
### OS/Platform

Windows

### Installed

Compiled from source

### Version

104.0.5112.102

### Tested upstream?

- [ ] I have tried reproducing this issue in Chrome and it could not be reproduced there
- [ ] I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there

### Description

No heavy ads block

### How to Reproduce?

Test with : 

 https://heavy-ads.glitch.me/

### Actual behaviour

Hello, I already talked about it but I thought it was a bug in Ungoogled chromium and not in chromium but I just discovered that this catastrophic bug also concerns chromium!

 I thought chromium blocked by default and natively heavy ads and crypto mining

https://blog.chromium.org/2020/05/resource-heavy-ads-in-chrome.html

 but I just discovered that chromium does not block heavy ads despite this option is implemented and integrated by default in chromium, and I tested with this site:

https://heavy-ads.glitch.me/

and regular classic version of chromium in woolyss site does not block heavy crypto ads and i test with microsoft edge browser and it blocks heavy ads, but Ungoogled chromium browser does not block heavy ads it is an important option and i don't see more interest to use chromium if tele important option does not work, please it will be great to activate this important option, thank you very much :)

### Expected behaviour

Block heavy ads

**Comment by PF4Public** [comment]
Is it a duplicate of #1944?

**Comment by trimechee** [comment]
No, beacause there is new information and possible solution about the bug and the bug was classified in  "discussion tag" however this is a big bug,  i said " I already talked about it but I thought it was a bug in Ungoogled chromium" but i discover this now it is bug in CHROMIUM BROWSER with API of chrome enabled , so i find possible solution and a line of thought, the bug is in chromium upstream, if this bug is resolved in chromium, it will be solved in ungoogled chromium , this new subject, i talk about chromiumbrowser, so the solution is to contact chromium tems developers to find why heavy ads are not blocked , this is strange , this feature seems to be reserved for mainstream browsers

**Comment by PF4Public** [comment]
> solution is to contact chromium tems developers to find why heavy ads are not blocked

Have you contacted them already?

**Comment by trimechee** [comment]
@PF4Public I do not have the technical skills to discuss with them, at leat chromium has this ad blocker enabled, but ungoogled chromium does not have this native ad block, why ? there will  bemigration to manifest v3 and many adblocker will be weak perhaps so native ad block may be useful

**Comment by j-james** [comment]
I'm not sure why the native adblocker is removed / not present, but at the current moment, it's far inferior to uBlock Origin. (I use Bromite with the native adblocker on mobile and the difference is very noticeable.)

I think the future situation with Manifest v3 is up in the air. A lot of discussion has happened over at #662: the consensus seems to be that maintaining a patch for Manifest v2 is a lot of work; there doesn't seem to be a consensus on whether the ungoogled project has enough manpower to do so. uBlock Origin's future is also uncertain going forward: I don't know whether they would maintain a version just for Brave (which is keeping v2) and Ungoogled Chromium. They probably would, there are a fair amount of Brave users and the maintenance burden would not be high.

tl;dr idk, but ublock origin's better for now

**Comment by j-james** [comment]
also, i agree that this is a duplicate / out of scope.
